fun/lms/views.py

```diff
@@ -0,0 +1,16 @@
+import re
+from django.conf import settings
+from django.core.urlresolvers import reverse
+from django.http import HttpResponse
+from django.shortcuts import redirect
+
+
+def richie(request, **kwargs):
+    """Extract params from route to redirect to the right Richie view"""
+    if (settings.PLATFORM_RICHIE_URL is not None):
+        return redirect("{richie_url:s}/{target:s}".format(
+            richie_url=settings.PLATFORM_RICHIE_URL,
+            target=kwargs.get("redirect_to", "")
+        ))
+
+    return redirect(reverse("dashboard"))
```

Comment:
or redirect to "redirecto_to" but on the current domain?
```suggestion
    return redirect("/{:s}".format(redirect_to))
```

Reply:
This should redirect often on a `404 Not found` page?

Reply:
Ok I see your point. It shouldn't happen the way we intend to use it as it redirects to a page from which you came. But I agree to leave it like this.